Makefile

```diff
@@ -0,0 +1,17 @@
+# If CNTR_PROG is undefined, then set it to be podman
+ifndef CNTR_PROG
+	CNTR_PROG = podman
+endif
+
+CNTR_USER=gvegayon
+IMAGE_NAME=cfa-epinow2-pipeline-dependencies:latest
+
+build:
+	$(CNTR_PROG) build -t $(IMAGE_NAME) -f Dockerfile-dependencies . && \
+	$(CNTR_PROG) tag $(IMAGE_NAME) $(CNTR_USER)/$(IMAGE_NAME)
```

Comment:
Why are you tagging the image again here? Why not just tag it on the first line?

I'm sure there's something clever going on here, but I'm not totally following it? Maybe document it in a comment?

Reply:
Also do you want to do the multi-stage build here?

Reply:
Nothing fancy there, just me playing with the code :P. It works for the moment, but it is not during the workflow. We could do a multi-stage build to lower the size of the image; like with caching more stuff, I would defer this for later when the build process is too much.